src/containers/CreateStreamMainCard/index.tsx

```diff
@@ -29,14 +36,24 @@ const INFINITY_DATE = new Date('Tue Oct 10 2100 00:00:00');
 
 const CreateStream = () => {
   const [isFormValidated, setIsFormValidated] = useState(false);
-  // const balances = useAppSelector((state) => state.user.info?.balances);
+  const [isOpenApproveModal, setIsOpenApproveModal] = useState(false);
+  const [isTokenAccessModal, setIsTokenAccessModal] = useState(false);
+  const [isWaitTransactionModal, setIsWaitTransactionModal] = useState(false);
+  const [isCreateStreamConfirmModal, setIsCreateStreamConfirmModal] = useState(false);
+  const [isWaitTransactionConfirmModal, setIsWaitTransactionConfirmModal] = useState(false);
+  const [isCompleteTransactionModal, setIsCompleteTransactionModal] = useState(false);
+  const [isOpenTransactionSuccessModal, setIsOpenTransactionSuccessModal] = useState(false);
```

Comment:
move all modals, states of the modals and everything related to them to the ConfirmTransaction component.